CHANGELOG.md

```diff
@@ -29,6 +29,12 @@ Markdown Spec](https://github.github.com/gfm/).
 ### Contributors
 -->
 
+## [v1.0.5] - Fix configuration for IIIF server
+[v1.0.6]: https://github.com/open-oni/open-oni/compare/v1.0.5...v1.0.6
+
+### Changed
+- Pinned to RAIS 4 for local development and updated configuration for that version
+
```

Comment:
Can we add the migration note from the dev PR?
> ### Migration
> - If you pinned a specific version of RAIS in your
  `docker-compose.override.yml`, make sure you either undo that or set up the
  appropriate environment, as RAIS 3 uses a slightly different configuration
  from RAIS 4.

I think it should go along with this, and I'll clean up the `dev` changelog when I merge `main` into `dev` :+1:

Reply:
Ah, good catch